email/1.3/modules/ROOT/edits/email-gmail.adoc

```diff
@@ -0,0 +1,160 @@
+= Connect to Gmail with Email Connector Examples - Mule 4
+:page-aliases: connectors::email/email-gmail.adoc
+
+To connect Anypoint Connector for Email (Email Connector) to your Gmail account, enable *Less Secure Apps* in your account and configure a global element for the server connection IMAPS, POP3S, or SMTPS you want to use.
+
+== Enable Less Secure Apps in your Gmail Account
+
+Google might block third-party mail clients if you do not enable the *Less Secure Apps* setting for your Gmail account, as Google blocks any apps that are not using security protocols that Google deems mandatory unless you allow less secure apps to access your Gmail account.
+
+When you try to connect to your Gmail for the first time, you will receive an email with a link for enabling less secure apps in your Gmail account.
```

Comment:
Ah! It would flow even more nicely if this sentence was first:
```suggestion
When you try to connect to your Gmail for the first time, you receive an email with a link for enabling less secure apps in your Gmail account. Be sure to do this, to prevent Google from blocking third-party mail clients that are not using security protocols that Google deems mandatory:
```
Also consider adding a caveat about audiences that have two-factor auth:
```suggestion
When you try to connect to your Gmail for the first time, you receive an email with a link for enabling less secure apps in your Gmail account. If your organization is not using two-factor authentication, be sure to do this, to prevent Google from blocking third-party mail clients that are not using security protocols that Google deems mandatory:
```